### DISCUSSION TITLE
Multiple Projects MLB + NBA + SOCCER -> Combined chat

### DISCUSSION THREAD

**Discussion by BGB2k11** [kyleskom/NBA-Machine-Learning-Sports-Betting#175] [Ideas]
Why dont we create a discord ? for the programmers ?

**Comment by Marcfeitosa** [comment]
great idea.
    Em quarta-feira, 5 de abril de 2023 às 03:25:19 BRT, BGB ***@***.***> escreveu:  
 
 


Why dont we create a discord ? for the programmers ?

**Reply by Skysafe64** [reply]
Great question

**Comment by phrygiantheory** [comment]
Something similar for Tennis would be amazing.

**Reply by Skysafe64** [reply]
> Something similar for Tennis would be amazing.

I'm already working on it

**Comment by phrygiantheory** [comment]
Sweet!!!  Can't wait.

**Comment by BGB2k11** [comment]
Server is:

https://discord.gg/BuCRQj7Kdk
